### PR TITLE
Added first draft for lose screen

### PR DESCRIPTION
**Pull request by GianiStatie**

### CODE REVIEW

File: Endscreen.gd [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	pass # Replace with function body.
+
+
+# Called every frame. 'delta' is the elapsed time since the previous frame.
+func _process(delta):
+	pass
```
Comment (Ratoone) [inline Endscreen.gd:11]: empty script

File: Main.gd [hunk 2]
```diff
@@ -8,6 +8,10 @@ extends Node2D
 func _ready():
 	for index in suitcase.suitcase.keys().size():
 		buildUI.update_tower_button(index, suitcase.get_name_by_index(index), suitcase.get_value_by_index(index))
+	
+	for child in units.get_children():
+		if child.is_in_group("chicken"):
+			SessionManager.number_of_chickens += 1
```
Comment (Ratoone) [inline Main.gd:14]: this might not be the best way to do it long term, but it works for now

Comment (GianiStatie) [reply Main.gd:14]: I know I know. We may want to spawn the chickens rather than have them in the scene in the future.

Comment (Ratoone) [reply Main.gd:14]: yes. we might also want to not rely on a session manager, but that's a different refactoring

**Review by Ratoone — APPROVED** [review]
i love the flavor text 😂

**Pull request merged**